stories/ag/Listings/DesktopListing.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { DesktopListing, SingleFamilyDesktopListing } from 'react-ui-ag/src'
+import { boolean } from '@storybook/addon-knobs'
+
+const baseListing = {
+  banners: ['New Construction', 'Sponsored'],
+  photos: [
+    {
+      path: 'imgr/2576db62ffa153ebef00317a5c68a368/',
+      caption: 'test 1',
+    },
+    {
+      path: 'imgr/d56984e959a3feb1235f85ee202a0fc6/',
+      caption: null,
+    },
+    {
+      path: 'imgr/fd972eb03a0463c484580349ad5177b7/',
+      caption: null,
+    },
+  ],
+  price: '$1170+',
+  bedrooms: '1-3 Beds',
+  bathrooms: '1-2 Baths',
+  unitLevelAvailability: '12 Units Available',
+  availability: 'Available Now',
+  name: 'The Grand at Beckwood',
+  address: '3921 Tugaloo River Drive Northside Park Washington Parkway',
+  rating: {
+    score: 4,
+    label: '(20)',
+    uniqueId: '123',
+  },
+  phone: '(404) 770-5555',
+}
+
+const props = {
+  listing: baseListing,
+  onClick: () => action('click')('listing cell click'),
+  favoriteButton: {
+    onClick: (listing, value) => action('click')('favorite toggled to ', value),
+    children: '♥',
```

Comment:
Seeing this always bugs me a little. I know we're going to replace it with an SVG in ag.js, but why can't `react-ui-ag` provide that out of the box?

Reply:
Not a blocker for the PR but I would like to see this addressed some day?

Reply:
I'm not sure why it bugs you.  This was an architecture decision.  How are you suggesting we add the svg in react-ui-ag, through a sprite, directly?   If the icon changes for some reason we now have to edit it here, publish and then bump the version which is seems somewhat superfluous.  We don't do this is this is exact reason. Any visual styling is generally done outside of the scope of react-ui. We can talk about this if you'd like if you have ideas on how we can separate these out and include icons and styling.